go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/coredns/coredns
 
-go 1.23.0
+go 1.24
 
-toolchain go1.24.1
+toolchain go1.24.3
```

Comment:
Please keep this version as 1.23.0. We want to be able to support the current Go releases, so we still want to keep 1.23.0 as the minimum supported version.